pkg/plugins/plugins.go

```diff
@@ -29,6 +29,7 @@ var (
 	StaticRoutes []*PluginStaticRoute
 	Apps         map[string]*AppPlugin
 	Plugins      map[string]*PluginBase
+	Errors       map[string]*PluginError
```

Comment:
Could maybe make this private to the package if you add a public function to retrieve list of errors.